econml/tests/test_rscorer.py

```diff
@@ -20,52 +20,109 @@ def _fit_model(name, model, Y, T, X):
 
 class TestRScorer(unittest.TestCase):
 
-    def _get_data(self):
+    def _get_data(self, discrete_outcome=False):
         X = np.random.normal(0, 1, size=(100000, 2))
         T = np.random.binomial(1, .5, size=(100000,))
-        y = X[:, 0] * T + np.random.normal(size=(100000,))
-        return y, T, X, X[:, 0]
+        if discrete_outcome:
+            eps = np.random.normal(size=(100000,))
+            log_odds = X[:, 0]*T + eps
+            y_sigmoid = 1/(1 + np.exp(-log_odds))
+            y = np.array([np.random.binomial(1, p) for p in y_sigmoid])
+            # Difference in conditional probabilities P(y=1|X,T=1) - P(y=0|X,T=0)

```

Comment:
Should this be
```suggestion
            # Difference in conditional probabilities P(y=1|X,T=1) - P(y=1|X,T=0)
```
?

Reply:
Yes, that was a typo on my end. Thanks for catching that!